src/components/Pages/Products/Products.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import './Products.scss';
+import CardProduct from '../../molecules/Card-Product/CardProduct';
+import Dropdown from '../../molecules/Productdropdown/Dropdown';
+import Button from '../../atoms/Button/Button';
+import { useTranslation } from 'react-i18next';
+import { updateTitle } from '../../../utils/utils';
+import { scrollTop } from '../../../utils/scrollutils';
+import Scrolltop from '../../molecules/ScrollTop/Scrolltop';
+import { GlobalContext } from '../../../Context/globalContext';
+const Products = (props) =>{
+    const [totalProductsList] = React.useState(require('../../../../server/products/index.get.json'));
+    const [displayProductList, setDisplayProductList] = React.useState(totalProductsList);
+    const [selectedCategory, setSelectedCategory] = React.useState("");
```

Comment:
Too much complex state handling.

Reply:
Upon discussion, will try to reduce the load a bit, as i will need some guidance regarding the same.